### ISSUE TITLE
Using the sore property. 

### ISSUE THREAD

**Issue by dsnowdon** [SortableJS/polymer-sortablejs#3]
How can this be done via polymer?

**Comment by hyyan** [comment]
@dsnowdon  Any luck with that ? I have opened another issue [here](https://github.com/SortableJS/polymer-sortablejs/issues/10)

**Comment by dsnowdon** [comment]
@hyyan no, but after seeing your link, I have more info to go off so i'm going to give it another go https://github.com/RubaXa/Sortable/wiki/Sortable-v1.0-%E2%80%94-New-capabilities#methods-for-sorting-acquisition-and-modification

**Comment by hyyan** [comment]
@dsnowdon great ,please let me know if you have found something on this issue

**Comment by bstmedia** [comment]
Please see issue #10 .. I just pasted some working code.
